.gitignore

```diff
@@ -1,6 +1,7 @@
 favicon.*
 .htconfig.php
 .htpreconfig.php
+config/autoconfig.php

```

Comment:
Do we need this since it's a temporary file meant to be deleted eventually?

Reply:
hm .. This is just in case that somebody forgets to delete it, so it cannot get accidentally pushed.

Reply:
Why not use the system temporary path then? base_path issues?

Reply:
Normally the `autoconfig.php` file is created before the first start. So I thought it's better to move it to the same directory where the .htconfig.php will get created. For the purpose of traceability
between the autoconfig.php and the .htconfig.php

What do you mean with the system temporary path and base_path issues?

Reply:
Do I have to change something in the .github file? Or is it OK for you?

Reply:
> What do you mean with the system temporary path and base_path issues?

Can the `autoconfig.php` be created and then included from the system temporary folder?

> Do I have to change something in the .github file? Or is it OK for you?

Yes, please revert the change.

Reply:
OK, I reverted the change.

But when should the `autoconfig.php` get created? For me, it is saved at a given standard docker-compose setting and will get used at first visit of the website and afterwards get deleted. So you don't need to do anything except setting the environment-variable the first time you start docker.

Reply:
Would this help you: https://github.com/friendica/friendica/pull/4855

Reply:
Hm.. This is another approach

I thought when docker dies or the complete system crashes, all of your environment variables (f.e. in your `docker-compose.yml` configuration file) gets lost. So when you just use the autoconfig and the given environment variables for the first time you start, you're save even when the system crashes, because you wrote them as a "normal" `.htconfig.php` next to your data on a mountpoint/volume.